README.md

```diff
@@ -1,22 +1,28 @@
 # Uber Rides Java SDK (Beta) [![Build Status](https://travis-ci.org/uber/rides-java-sdk.svg?branch=master)](https://travis-ci.org/uber/rides-java-sdk)
 This SDK helps your Java App make HTTP requests to the Uber Rides API.
 
+## Prerequisites
+- Dependency manager: Gradle or Maven
```

Comment:
Alright, if it can be resolved with ivy or others, we could remove it. I just thought it might add clarity.

In regards to the reference, I think some developers might be not experts in Java, especially dep managers. It just felt like a barrier that could just be removed with this little note :)